falkordb-cluster/cluster-entrypoint.sh

```
@@ -102,7 +101,8 @@ log() {
 
 get_host() {
   local host_idx=$1
-  echo "$RESOURCE_ALIAS-$host_idx.$EXTERNAL_DNS_SUFFIX"
+  local deployment_mode=$(if [[ "$IS_MULTI_ZONE" == "1" ]]; then echo "mz"; else echo "sz"; fi)
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)